sim/adversary/adversary.go

```diff
@@ -14,6 +15,8 @@ type Host interface {
 	// Note that the adversary can subsequently delay delivery to some participants,
 	// before messages are actually received.
 	BroadcastSynchronous(sender gpbft.ActorID, msg gpbft.GMessage)
+	// Sends a message to all other participants.
+	Broadcast(*gpbft.GMessage)
```

Comment:
Nit: suggest listing the "normal" broadcast before the special one. These two could possibly be collapsed into one method with another parameter, too.